src/ui.sh

```diff
@@ -20,8 +20,10 @@ ui::fzf() {
 ui::select() {
    local -r cheats="$1"
 
+   [[ "$SHELL" =~ 'fish' ]] || local -r sub='$'
```

Comment:
Maybe ${SHELL:-}?

I'm not sure if there's a case where this variable may be unset.

Reply:
If it's unset, then it is most probably not fish shell, so don't need to provide a default value since it is double-quoted anyway.